py/pylya/cf.py

```diff
@@ -93,19 +94,20 @@ def cf(pix):
 def fast_cf(z1,r1,w1,d1,z2,r2,w2,d2,ang,same_half_plate):
     wd1 = d1*w1
     wd2 = d2*w2
-    rp = abs(r1-r2[:,None])*sp.cos(ang/2)
+    if rp_min<0 : rp = (r1-r2[:,None])*sp.cos(ang/2)
```

Comment:
because the computation of this correlation function is longer and for the moment I haven't modify the other tools ( dmat, do_fit, plotter...) so I want to be able to choose if I want abs() for the x-correlation